backend/persistent/datastores/Athena.js

```diff
@@ -0,0 +1,189 @@
+
+import {executeQuery} from './drivers/AWSAthenaDriver';
+
+import Logger from '../../logger';
+const SHOW_TABLES_QUERY = 'SHOW TABLES';
+const SHOW_SCHEMA_QUERY = 'SELECT table_name, column_name, data_type FROM '
+    + 'information_schema.columns WHERE table_schema ';
+const DEFAULT_QUERY_TIMEOUT = 2000;
```

Comment:
This default timeouts seem to short.

Not sure if it's the same, but [this documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Athena.html) quotes a default timeout of 2 minutes.